core/src/main/scala/cats/data/XorT.scala

```diff
@@ -219,6 +219,11 @@ trait XorTFunctions {
 
   final def pure[F[_], A, B](b: B)(implicit F: Applicative[F]): XorT[F, A, B] = right(F.pure(b))
 
+  /**
+   * Alias for XorT.right
+   */
+  final def liftT[F[_], A, B](fb: F[B])(implicit F: Functor[F]): XorT[F, A, B] = right(fb)
```

Comment:
Without the implicit evidence for a `Functor` it won't compile since `right` needs one.